legal/discord-rules.php

```diff
@@ -0,0 +1,66 @@
+<?php
+require("../template/top.php");
+head("Discord Rules", true);
+?>
+<style>
+    h3 {
+        margin-top: 20px;
+    }
+</style>
+
+<section class="breadcrumb-classic">
+    <div class="shell">
+        <ul class="list-breadcrumb">
+            <li><a href="/">Home</a></li>
+            <li><a href="/legal">Legal</a></li>
+            <li>Discord Rules</li>
+        </ul>
+    </div>
+</section>
+
+<section class="section-50 section-md-75 section-lg-100">
+    <div class="shell text-sm-left">
+        <div class="range">
+            <div class="cell-xl-12">
+                <div>
+                    <h2>Discord Rules</h2>
+                    <p>The following set of rules govern communications in the official UNT Robotics Discord Guild/Server.</p>
+                    <h3>Access</h3>
+                        <ol>
+                            <li>The UNT Robotics Discord server is open to all current UNT students.</li>
+                            <li>UNT Robotics alumni are permitted to remain part of the server after their graduation.</li>
+                            <li>UNT Faculty are permitted to join and use the server.</li>
+                            <li>Any non-UNT students or faculty must be vetted before joining the server. All non-UNT affiliated members must first complete all appropriate paperwork with UNT to be allowed to work with students. The main criteria for non-UNT affiliated members are as follows:</li>

```

Comment:
Link to relevant paperwork

Reply:
This is intentionally vague, we can't provide the appropriate paperwork for all situations and by just specifying that the right paperwork must be filled out and verified by us, we can ensure nothing slips through the gaps.